crm_compassion/model/event_compassion.py

```diff
@@ -78,8 +81,12 @@ class event_compassion(models.Model):
         compute='_set_analytic_lines', readonly=True, store=True)
     balance = fields.Float(
         compute='_set_analytic_lines', readonly=True, store=True)
+    number_reserve_children = fields.Integer(

```

Comment:
Rename `reserve` by `allocate` because `reservation` is a term that is already used for something else.